composer.json

```diff
@@ -34,6 +39,7 @@
         "rector/rector": "^0.14.3",
         "roave/infection-static-analysis-plugin": "^1.16",
         "spatie/phpunit-watcher": "^1.23",
+        "squizlabs/php_codesniffer": "^3.7",
```

Comment:
It's probably used in your projects, right? Don't remember that it was used in our repos.

Reply:
it is dev dependency, for development. Do i need remove it?

Reply:
yes sure